app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -7,22 +7,38 @@ class AfterSignInController < CandidateInterfaceController
     def interstitial
       current_candidate.update!(course_from_find_id: nil)
 
-      if current_application.submitted?
+      if current_application.submitted? && !current_application.continuous_applications?
         redirect_to candidate_interface_application_complete_path
       elsif current_application.contains_course?(course_from_find)
         flash[:warning] = "You have already selected #{course_from_find.name_and_code}."
-        redirect_to candidate_interface_course_choices_review_path
+        redirect_to course_choices_page
       elsif current_application.maximum_number_of_course_choices?
         flash[:warning] = I18n.t('errors.messages.too_many_course_choices', course_name_and_code: course_from_find.name_and_code)
 
-        redirect_to candidate_interface_course_choices_review_path
+        redirect_to course_choices_page
       else
-        redirect_to candidate_interface_course_confirm_selection_path(course_from_find.id)
+        redirect_to confirm_selection_page
```

Comment:
Nope, this is the direct redirect interstitial coming from Find.

Reply:
Something like these from my personal notes when I was trying to understand the current implementation
```
   1
   2 -> Add course from find /apply?providerCode=1&courseCode=2
   3     This can redirect the candidate:
   4         1. To sign if they are not login
   5         2. To this interstitial action if everything is ok
   6         3. Not found
   7 -> interstitial action
   8     This can redirect the candidate:
   9         1. The your applications tab with an error message
  10           a. Course already added error if already added course
  11           b. Too many course choices if pass the maximum
  12         2. The confirm selection page if is ok
```

Reply:
Sorry, I miss read this line.
https://github.com/DFE-Digital/apply-for-teacher-training/pull/8519/files/7448f529b48d9b1b2111b96ee7709c01ac249323#diff-2a0d49b03caf5dd052ebc907935d9d6d3a9b4fe6ecde0922017f8c297d294c30R10